core/src/main/java/org/apache/accumulo/core/file/rfile/MetricsGatherer.java

```diff
@@ -55,12 +54,8 @@
    *
    * @param key
    *          Key object of the entry you are collecting metrics from
-   *
-   * @param val
-   *          Value object of the entry you are collecting metrics from
-   *
    */
-  void addMetric(Key key, Value val);
```

Comment:
Not sure about this one. Although not public API, the original intent was to make this flexible/pluggable. Another implementation could use it.

Reply:
No problem, I'll revert.